1904-11-03.xml

```diff
@@ -0,0 +1,2652 @@
+                        <div type="item">
+                            <head> PROFITABLE PASTIME.-</head>
+                            <p>It is not only the wish to learn quickly which urges so many to go to
+                                the Berlitz Schools, but also the prospect of taking an hour's
+                                lesson in the most agreeable way. For example, many ladies take
+                                lessons simply for pleasure a very useful pastime. A trial lesson in
+                                English, French, or any other language may be had at Alexandria, 26,
+                                Rue Eglise Copte, or at Cairo, 1, Sharia Kamel. [ADVT.]</p>
+                        </div>
+                        <div type="item">
+                            <head>THE PYRAMIDS.-</head>
+                            <p>Visitors to the Pyramids will be pleased to learn that an attempt is
+                                being made to meet the requirements of the "inner man" in this
+                                locality. A bar and restaurant have been opened where tea, etc., can
+                                be obtained at moderate prices. All the refreshments are supplied by
+                                Mr. Spathis, the well-known merchant and mineral water manufacturer,
+                                and this is a sufficient guarantee of their high quality. </p>
+                        </div>
+                        <div type="item">
+                            <head/>
+                            <p>THE NEW NORDDEUTSCHER LLOYD'Imperial Mail steamer Prinz Eitel
+                                Friedrich, a description of which we, gave in a recent issue, left
+                                Suez yesterday morning, bound to China. She had a full first and
+                                second-class passenger list, which included 17 naval officers, 32
+                                cadets, and 10 non-commissioned officers, reliefs for the German
+                                squadron in the Far East. There were also on board 17 Belgian
+                                soldiers in charge of an officer, bound for China. </p>
+                        </div>
+                        <div type="item">
+                            <head/>
+                            <p>H.M. TRANSPORT Soudan (Capt. S. de B. Lockyer, R.N.R.) arrived at
+                                Port Said on Tuesday from Bombay for Malta and Southampton. There
+                                were 1,362 of all ranks on board, including the Cheshire Regiment,
+                                numbering 427 officers and men, who are to be stationed at Lichfield
+                                on their arrival in England. Sixty-nine details are to land at
+                                Malta. The officer commanding the troops is Lt. Col. Hardy, Cheshire
+                                Regt. </p>
+                        </div>
```

Comment:
it looks like this is the end of the local and general section--if so add a </div> to close the whole section